pages/Account.qml

```diff
@@ -192,6 +192,14 @@ Rectangle {
                         Layout.fillWidth: true
                         color: "transparent"
 
+                        Rectangle {
+                            visible: itemMouseArea.containsMouse || index === currentAccountIndex
+                            height: subaddressAccountListRow.subaddressAccountListItemHeight
+                            width: parent ? parent.width : undefined
+                            Layout.fillWidth: true
+                            color: MoneroComponents.Style.titleBarButtonHoverColor
+                        }
```

Comment:
Can't we use `tableItem2` `color` instead of introducing another rectangle? Like `color: itemMouseArea.containsMouse || index === currentAccountIndex ? MoneroComponents.Style.titleBarButtonHoverColor : "transparent"`

Reply:
It seems the obvious solution, but I tried this, and it doesn't work.

Reply:
hmm i'll see if it works. just a moment

Reply:
I decided to add a new rectangle because Mouse area has two properties that deal with mouse hover : 
```
onEntered: tableItem2.color = MoneroComponents.Style.titleBarButtonHoverColor
onExited: tableItem2.color = "transparent"
```

and using the following code doesn't work:
```
onEntered: tableItem2.color = (index === currentAccountIndex ? MoneroComponents.Style.titleBarButtonHoverColor : "transparent")
onExited: tableItem2.color = (index === currentAccountIndex ? MoneroComponents.Style.titleBarButtonHoverColor : "transparent")
```

Reply:
> It seems the obvious solution, but I tried this, and it doesn't work.
> I decided to add a new rectangle because Mouse area has two properties that deal with mouse hover and change tableItem2.color value:

Don't add new rectangle. Remove Mouse area `onEntered` and `onExited` handlers.
The approach i suggest works pretty well.

```diff
diff --git a/pages/Account.qml b/pages/Account.qml
index 49c06156..d0bd5e43 100644
--- a/pages/Account.qml
+++ b/pages/Account.qml
@@ -190,7 +190,9 @@ Rectangle {
                         height: subaddressAccountListRow.subaddressAccountListItemHeight
                         width: parent ? parent.width : undefined
                         Layout.fillWidth: true
-                        color: "transparent"
+                        color: itemMouseArea.containsMouse || index === currentAccountIndex
+                            ? MoneroComponents.Style.titleBarButtonHoverColor
+                            : "transparent"
 
                         Rectangle {
                             color: MoneroComponents.Style.appWindowBorderColor
@@ -226,7 +228,9 @@ Rectangle {
 
                             MoneroComponents.Label {
                                 id: nameLabel
-                                color: MoneroComponents.Style.dimmedFontColor
+                                color: index === currentAccountIndex
+                                    ? MoneroComponents.Style.defaultFontColor
+                                    : MoneroComponents.Style.dimmedFontColor
                                 anchors.verticalCenter: parent.verticalCenter
                                 anchors.left: idLabel.right
                                 anchors.leftMargin: 6
@@ -276,11 +280,10 @@ Rectangle {
                             }
 
                             MouseArea {
+                                id: itemMouseArea
                                 cursorShape: Qt.PointingHandCursor
                                 anchors.fill: parent
                                 hoverEnabled: true
-                                onEntered: tableItem2.color = MoneroComponents.Style.titleBarButtonHoverColor
-                                onExited: tableItem2.color = "transparent"
                                 onClicked: {
                                     appWindow.currentWallet.switchSubaddressAccount(index);
                                     if (selectAndSend)
```

Reply:
Yes you're right, I tested it here. I'm just doing some modifications now, will updated new code soon.